src/main/java/com/epam/training/olha_haichenkova/task_3/page/PricingCalculatorPage.java

```diff
@@ -0,0 +1,198 @@
+package com.epam.training.olha_haichenkova.task_3.page;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class PricingCalculatorPage extends AbstractPage{
+
+    private static final String PRICING_CALCULATOR_URL = "https://cloud.google.com/products/calculator";
+    static final String OUTER_IFRAME_XPATH = "//main//devsite-iframe/iframe";
+    static final String INNER_IFRAME_ID = "myFrame";
+
+    @FindBy(xpath = "//md-tab-item[@id='tab-item-1']")
+    private WebElement computeEngineTab;
+
+    @FindBy(xpath = "//input[@ng-model='listingCtrl.computeServer.quantity']")
+    private WebElement numberOfServersInput;
+
+    @FindBy(xpath = "//md-select[@ng-model='listingCtrl.computeServer.os']//span[@class='md-select-icon']")
+    private WebElement operatingSystemSoftwareSelectIcon;
+
+    @FindBy(xpath = "//md-option[@value='free']")
+    private WebElement operatingSystemSoftwareOption;
+
+    @FindBy(xpath = "//md-select[@ng-model='listingCtrl.computeServer.family']//span[@class='md-select-icon']")
+    private WebElement vMFamilySelectIcon;
+
+    @FindBy(xpath = "//md-option[@value='gp']")
+    private WebElement vMFamilyOption;
+
+    @FindBy(xpath = "//md-select[@name='series']//span[@class='md-select-icon']")
+    private WebElement vMSeriesSelectIcon;
+
+    @FindBy(xpath = "//md-option[@value='n1']")
+    private WebElement vMSeriesOption;
+
+    @FindBy(xpath = "//md-select[@ng-model='listingCtrl.computeServer.instance']//span[@class='md-select-icon']")
+    private WebElement machineTypeSelectIcon;
+
+    @FindBy(xpath = "//md-option[@value='CP-COMPUTEENGINE-VMIMAGE-N1-STANDARD-8']")
```

Comment:
Looks like too specific selector. We should be able to choose another options from the list